### ISSUE TITLE
What happens if the job is removed while datasets refer to it in their

### ISSUE THREAD

**Issue by advanderveer** [nerdalize/nerd#278]
Whenever a job is removed while the dataset is referring to it "input from"/'output for" we might want to dereference this.

**Comment by LilliJane** [comment]
I don't know if we should dereference this or just indicate that the job was removed (like with a star at the end of the job name ? `j-gnt8l*`. Because if we dereference this, we will have some datasets without any information about jobs and I find it quite disturbing.

**Comment by advanderveer** [comment]
I agree that removing the reference is probably confusing, if we decide to fetch the pods as part of the dataset listing (to see if the referenced job still exists) it might be worthwhile to also use this to fetch the state of the job and add it in the column. This can then also be used to indicate it was deleted. So like this (with proper columen spacing):

```
DATASET   CREATED AT       SIZE     INPUT FOR   OUTPUT FROM   
d-hf7k7   28 minutes ago   0 B                  j-j7kj8:running       
d-v6fdd   28 minutes ago   409 kB   j-j7kj8:deleted                   
d-fmmp2   2 days ago       2.3 kB   j-vv46m:completed                   
```

**Comment by LilliJane** [comment]
This issue is also related to #355 #306
